integration_tests/integration/e2e/legalSenderJourneyE2e.spec.ts

```diff
@@ -14,13 +13,14 @@ context('Legal Sender Journey E2E', () => {
     cy.task('stubRequestLink')
     cy.task('stubVerifyLink')
     cy.task('stubCreateBarcode')
-  })
 
-  it('should allow Legal Senders to perform all actions as part of their workflow', () => {
     // Request a magic link and click it to sign in and land on the Find Recipient By Prison Number page
     cy.visit('/link/request-link')
     Page.verifyOnPage(RequestLinkPage).submitFormWithValidEmailAddress('valid@email.address.cjsm.net')
     cy.visit('/link/verify-link?secret=a-valid-secret')
+  })
+
+  it('should allow Legal Senders to perform all actions as part of their workflow - images', () => {
```

Comment:
Do we want an E2E that does all of this, but is for the PDF coversheet journey?

Reply:
I did that at first. But it seemed a waste of time as it just repeats all the same steps. The PDF covesheet journey is also tested elsewhere - hence I left it like this.

Reply:
ok 👍